### ISSUE TITLE
Cancer cell fraction

### ISSUE THREAD

**Issue by fpbarthel** [gavinha/TitanCNA#70]
Hi @gavinha,

This seems to make sense to me, but not sure if this was done before and/or makes sense to you. But could one get the cancer cell fraction of a somatic mutation by taking the product of the VAF and the local cellular prevalence as given by TITAN?

Best,
Floris

**Comment by gavinha** [comment]
Hi @fpbarthel 

For cancer cell fraction estimates of SNVs, I recommend you check out PyClone and PhyloWGS (and other similar tools). Their corresponding papers will have descriptions of how they compute this value.

Subclonal structures between copy number and SNVs may be different. Therefore, I am not sure if you can necessarily use TITAN's cellular prevalence this way.

Best,
Gavin

**Comment by fpbarthel** [comment]
Yeah, thanks for your response. I quickly figured this would only make sense for diploid regions, and there is no point in TITAN to determine CP of diploid regions. Can the SNV estimates be related to the CNV estimates, eg. SNV estimate from PyClone versus CNV estimates from TITAN to infer relative timing?
